.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I would move this to `react` so it matches the eslint config

Reply:
But the prettier config is not necessarily only used on the `react/` directory. It can also be used to format other dirs (like `node`, etc), `json` files, etc.